estuary/models/koji.py

```diff
@@ -15,6 +15,7 @@
 class KojiBuild(EstuaryStructuredNode):
     """Definition of a Koji build in Neo4j."""
 
+    __label_display__ = 'Koji build'
```

Comment:
I'm thinking we should make this just `build` instead of `Koji build` but I could be swayed either way. @yashvardhannanavati opinions on this?